File: VB/WindowServiceExample/ViewModels/WindowViewModel.vb
Imports DevExpress.Mvvm
Imports DevExpress.Mvvm.POCO

Namespace WindowServiceExample.ViewModels
    Public Class WindowViewModel
        Protected Sub New()
        End Sub

        Public Shared Function Create() As WindowViewModel
            Return ViewModelSource.Create(Function() New WindowViewModel())
        End Function
    End Class
End Namespace

File: CS/WindowServiceExample/ViewModels/WindowViewModel.cs
using DevExpress.Mvvm;
using DevExpress.Mvvm.POCO;

namespace WindowServiceExample.ViewModels {
    public class WindowViewModel {
        protected WindowViewModel() { }

        public static WindowViewModel Create() {
            return ViewModelSource.Create(() => new WindowViewModel());
        }
    }
}
